Mathlib/Order/Bounds/Basic.lean

```diff
@@ -1702,17 +1705,31 @@ The dual notion
 
 does not appear to play a significant role in the literature, so is omitted here.
 -/
-def ScottContinuous (f : α → β) : Prop :=
-  ∀ ⦃d : Set α⦄, d.Nonempty → DirectedOn (· ≤ ·) d → ∀ ⦃a⦄, IsLUB d a → IsLUB (f '' d) (f a)
-#align scott_continuous ScottContinuous
+def DScottContinuous (D : Set (Set α)) (f : α → β) : Prop :=
+  ∀ ⦃d : Set α⦄, d.Nonempty → DirectedOn (· ≤ ·) d → d ∈ D → ∀ ⦃a⦄, IsLUB d a → IsLUB (f '' d) (f a)
+#align scott_continuous DScottContinuous
```

Comment:
Shouldn't this keep about `ScottContinuous`?

Reply:
Yes, done.